src/view/prDocumentCommentProvider.ts

```diff
@@ -64,6 +64,28 @@ export class PRDocumentCommentProvider implements vscode.DocumentCommentProvider
 
 		return await this._prDocumentCommentProviders[params.prNumber].replyToCommentThread(document, range, commentThread, text, token);
 	}
+
+	async editComment(document: vscode.TextDocument, comment: vscode.Comment, text: string, token: vscode.CancellationToken): Promise<void> {
+		let uri = document.uri;
+		let params = fromPRUri(uri);
+
+		if (!this._prDocumentCommentProviders[params.prNumber]) {
+			throw new Error(`Couldn't find document provider`);

```

Comment:
We are returning null in other places when lookup fails. But this is a void method. Should we just ignore the call, or throw an error?

Reply:
We should throw, but can we make the error message actionable to the user?

Reply:
we should throw an error

Reply:
we should throw an error

Reply:
throw an error

Reply:
throw an error

Reply:
throw `Error`

Reply:
throw `Error`

Reply:
throw `Error`

Reply:
throw `Error`